content/client-server-api/modules/event_replacements.md

```diff
@@ -133,13 +133,6 @@ being overwritten entirely by `m.new_content`, with the exception of `m.relates_
 which is left *unchanged*. Any `m.relates_to` property within `m.new_content`
 is ignored.
 
-{{% boxes/note %}}
-Note that server implementations must not *actually* overwrite
-the original event's `content`: instead the server presents it as being overwritten
```

Comment:
We might want to stick the first part of this note somewhere still, just in case servers do super strange things with their events? 

Possibly at the end of the server implementation section with a bit more detail?
```
{{% boxes/note %}}
Note that server implementations must not *actually* overwrite
the original event's `content`. Doing so would cause the original
event to fail signature checks.
{{% /boxes/note %}}

Reply:
This feels like it is more confusing than helpful. As a server implementer, I am led to wonder wtf this "*actually*" means.

I have added some more words to the "server-side aggregation" section which might help here (https://github.com/matrix-org/matrix-spec/pull/1440/commits/97b7544f2c212ef19323c1f835607e3b57331593#diff-7791fd17e89af8337a50aaa7e5fdf6cb1c20013d38cd491cda852e696884be76R248).